gms/factories/src/main/java/com/linkedin/gms/factory/search/SearchServiceFactory.java

```diff
@@ -18,6 +19,7 @@
   private ElasticSearchService elasticSearchService;
 
   @Bean(name = "searchService")
+  @Primary
```

Comment:
just curious, why is this needed?

Reply:
I am not sure :( Build failed because it said there were two instances of SearchService one from here and one from ElasticSearchServiceFactory. I have no idea why this is like this while graph service works fine.